mkdocs.yml

```diff
@@ -671,7 +671,7 @@ nav:
           - Adding and Updating Nautobot Settings: "development/core/settings.md"
           - Advanced Docker Compose Usage: "development/core/docker-compose-advanced-use-cases.md"
           - Application Registry: "development/core/application-registry.md"
-          - Bootstrap UI: "development/core/bootstrap-ui.md"
```

Comment:
Might want to add a redirect from this path to the new file, somewhere around line 297?